swagger.json

```diff
@@ -2574,9 +2574,234 @@
                     }
                 }
             }
+        },
+        "/addresses": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Fetch all addresses/geofences for the organization",
+                "operationId": "GetOrganizationAddresses",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "List of addresses/geofences. For each address entry, the geofence field will only be populated with either 'circle' or 'polygon', depending on the geofence type.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#definitions/Address"
```

Comment:
I believe this should be `#/definitions/Address`. Is this rendering correctly on editor.swagger.io?

Reply:
This looks to be rendering fine, but I'll update it anyways.

Reply:
Nevermind, it was taking the errors a minute to load at the top. I see the error message for this.